app/views/admin/settings/index.html.erb

```diff
@@ -22,7 +22,16 @@
                                 <%=link_to setting.code, admin_setting_path(setting) %>
                             </td>
                             <td><%= setting.description %></td>
-                            <td><%= setting.value %></td>
+                            <td>
+                              <% if filled_json_array?(setting.value)
+                                values = JSON.parse(setting.value)
+                                values.each do |value| %>
+                                  <a class='ui label'><%= value %></a>
+                              <% end
```

Comment:
Does this work this way? (`else` without `if`?)
Can we extract logic from view?

Reply:
Actually `if` was present, but doesn't really matter because I'm moving whole view logic to helper instead.

Reply:
But it was closed right before `else` https://github.com/internetee/auction_center/blob/6f6215a30828d46b816dbfc90635fa44609fee34/app/views/admin/settings/index.html.erb#L30

Reply:
Internal `do` loop was closed with that `end`, but indentation was wrong, indeed. Seems like `html.erb` doesn't really care about that.